roles/bind-sync-app-apb/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- name: "Retrieve route to sync"
+  shell: "oc get routes/sync-app -n '{{ namespace }}' -o jsonpath='{.spec.host}'"
```

Comment:
this probably will not work if there are multiple instances of the sync app

Reply:
@wei-lee That is part of a different JIRA. (See my multi binding branch)